cxx/distributions/bigram_test.cc

```diff
@@ -0,0 +1,38 @@
+// Apache License, Version 2.0, refer to LICENSE.txt
+
+#define BOOST_TEST_MODULE test Bigram
+
+#include <boost/test/included/unit_test.hpp>
```

Comment:
This reminds me, at some point we should clean up `test_misc` and remove what's now redundant in the unit tests.